src/panoptes/pocs/camera/camera.py

```diff
@@ -221,6 +223,10 @@ def temperature_tolerance(self, temperature_tolerance):
             temperature_tolerance = temperature_tolerance * u.Celsius
         self._temperature_tolerance = temperature_tolerance
 
+        # Wait for temperature to stabilise
+        if self.cooling_enabled:
+            self.wait_for_stable_temperature(blocking=False)
+
```

Comment:
It's probably an unnecessary addition. I put it in because if `temperature_tolerance` changes, the definition of temperature stability also changes. So a camera temperature may be stable at a given (high) tolerance, but could be regarded as unstable in the tolerance is reduced.

Reply:
But does changing that tolerance cause the cooling to automatically adjust?

I'm fine leaving it in here but maybe make a comment that it's there as a backup but is not primarily responsible for adjusting.